Move New England mailing list, add New England organizer guide

UnsolvedCypher opened the pull request:
This moves the New England mailing list to /newengland/mailing, adds a small landing page to /newengland, and adds an organizer guide to /newengland/organizing for local Delegates to be able to refer newer organizers to. [Here's](https://www.matthewmcmillan.me/delegate/) what the content is like.

UnsolvedCypher committed:
move New England mailing list, add New England organizer guide

UnsolvedCypher committed:
replace tabs with spaces

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="the-venue">The venue</h3>
+                <p>The venue is a key part of the competition experience. The organizer is entirely responsible for choosing and booking the venue. Of course, the Delegate needs to approve the venue (do NOT book without explicit Delegate approval). In general, you should look for a venue that has</p>
+                <ul>
+                        <li>enough space for everyone (there is usually extra person per competitor, so the room should be able to easily hold double the competitor limit)</li>
```
there is usually *one* extra person per competitor

(is that right?  I thought it was usually more like 1.5 or 2 guests on average.

UnsolvedCypher replied:
When I last checked, it was about 1. As the competitor get younger, I assume this goes up, so I'll check more recent competitions for better statistics.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                        <li>a reasonable price (this will be discussed later)</li>
+                        <li>a PA system (nice to have, but not a requirement)</li>
+                        <li>a projector (even less of a requirement but still nice to have)</li>
+                        <li>tables (rectangular competition tables are a must (or you’ll need to rent some), and other tables for the audience and competitors to sit at are nice)</li>
```
There's too many parentheses in these bullets, definitely get rid of the nested parentheses, and see if you can get rid of some of the others as well.  Maybe break this list into a "requirements" and "nice to have" if that helps.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                </ul>
+                <p>When you start writing up a budget, you should have a good idea of how much the venue will cost. Schools and universities can often provide low-cost venues, but that’s not always an option. Be sure to make note of required custodians or police officers, as this usually costs extra. Places of worship like churches may be good candidates, but their space is often either too small ...
+                <h3 id="pa-system-wifi">PA System, WiFi</h3>
+                <p>A PA system is really a must. If the venue doesn’t have one, Matthew McMillan has one he may be able to bring, but he needs to know this in advance. It’s very difficult to have a competition where all announcements are made by shouting. Similarly, it’s possible to have a competition without WiFi, but it’s far from ideal. If this has to happen, let your Delegate know in advance. ...
```
It's a bit confusing that it says it's "nice to have" above, then changes to "really a must", then back to not required.

Reword this bullet as "It's preferable to use a PA system in the venue.  If the venue doesn't have one, bring this up with your delegate, and we may be able to find an alternate arrangement."

timreyn replied:
"really a must" is also very colloquial.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="coordinating-staff">Coordinating Staff</h3>
+                <p>We have a template of a Google Form we use to collect staff applications which the Delegate will set up. While we’re happy to provide guidance, the organizer is responsible for providing lunch to the staff and properly coordinating with all staff members.</p>
+                <p>Staff typically have perks of free registration and free lunch in exchange for helping out all day. Staff must pay to register and will be refunded after the competition, as an incentive to actually do what is expected of them. Staff applications are posted on the website to allow anyone to fill out a form and apply for staff, but it’s up to our discretion to vet these people a ...
+                <p>Additionally, at recent competitions, we have been doing what is known as the “Hungarian System” where all competitors are assigned judging roles. This helps people become familiar with judging, and reduces the need for asking for volunteers constantly. It’s great to have a competitor tutorial at the beginning of the competition to help people understand how judging works, and  ...
```
Can we drop the term "hungarian system"?  I think people haven't been calling it that much recently, especially now that it's built into groupifier groups.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="pa-system-wifi">PA System, WiFi</h3>
+                <p>A PA system is really a must. If the venue doesn’t have one, Matthew McMillan has one he may be able to bring, but he needs to know this in advance. It’s very difficult to have a competition where all announcements are made by shouting. Similarly, it’s possible to have a competition without WiFi, but it’s far from ideal. If this has to happen, let your Delegate know in advance. ...
+                <h3 id="the-finances">The finances</h3>
+                <p>The organizer is responsible for creating a budget, though the Delegate will need to approve it. The organizer will usually need to create a Stripe account to collect the registration fees and link this to the WCA website, but I’m happy to do this if that poses a problem. Please fill out the budget tab in the spreadsheet linked to at the top of this document when it’s time to m ...
```
Get rid of the word "I'm" (also check for other uses of first-person, since this isn't just one person)
Also, can you use the words "net income" rather than "profit"?

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <p>There is still work to be done after the competition is announced! During this time it is imperative that you regularly check your email and respond to competitors and the delegate as many things are time sensitive.</p>
+                <h3 id="registration-competitor-limits-and-wait-lists">Registration, competitor limits, and wait lists</h3>
+                <p>Registration is done on the WCA website. When a competitor pays, they will need to be manually accepted by the organizer or Delegate to actually be registered. Registration should generally open as soon as possible, but it’s good to leave some time after the competition is announced just for people to be able to find out about it before they have to register.</p>
+                <p>Competitor limits are a big deal. They are part of the WCA regulations. No competitors can be accepted when the limit is reached, no matter what, so be extra careful when accepting registrations. This means you should take extra care in deciding the competitor limit. You should plan for twice the number of people at the competition as there are competitors (so 200 total people  ...
```
This point is wordy.  Consider rewording as:

You will need to set a competitor limit.  When deciding on the competitor limit, remember to account for spectators and family members as well.  You <strong>cannot</strong> accept any competitors after the competitor limit has been reached.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="lunch">Lunch</h3>
+                <p>You may choose to provide lunch for the competitors and their families. You should really consider doing this if the competition is in a location where there aren’t very many places to get a quick lunch. Pizza is usually easiest to coordinate, and not too expensive. Keep in mind that you’ll want to talk to the pizza place well in advance to see if they have bulk pricing and to  ...
+                <h3 id="cubecomps">CubeComps</h3>
+                <p>CubeComps is the live results system currently used just about everywhere. This will need to be set up after the competition is announced. You should email <a href="mailto:admins@cubecomps.com">admins@cubecomps.com</a> after it’s announced to get this set up. There is also some setup work involved in actually importing your registration file, and you’ll need to set up rounds an ...
```
This line's wordy and colloquial.

CubeComps is the main results system for WCA competitions.  Send an email to <email> after the competition is announced to request a CubeComps page.  You'll also need to set up the events and rounds for the competition, and import the final registration list.

Also, I don't think it's worth saying "ask the delegate if you need help" -- that should be the default on this whole thing.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="sponsorship">Sponsorship</h3>
+                <p>Many competitions are sponsored by cube stores, who will provide prizes and sometimes even vend at the competition. You should try to figure this out before announcing.</p>
+                <h3 id="cubingusa-supported-competitions">CubingUSA Supported Competitions</h3>
+                <p>CubingUSA is supporting some competitions. Here’s a brief summary from their website:</p>
```
Can we drop this and just link to the supported comps page?  The line about the reason for doing this may be useful.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <p>In CubeComps, after your email <a href="mailto:admins@cubecomps.com">admins@cubecomps.com</a> to create a site for your competition, you can log in as the administrator and upload the csv. You’ll need to set cutoffs for all of the events with cutoffs so that they are processed correctly. You’ll also need to set the number of advancing competitors. Let us know if you need help w ...
+                <h3 id="scorecards-and-certificates">Scorecards and certificates</h3>
+                <p>The organizer is responsible for generating and printing scorecards as well as designing and printing the certificates. <a href="https://jonatanklosko.github.io/groupifier/">Groupifier</a> is typically used for scorecard generation of first round cards and CubeComps is used to generate blank cards. Groupifier is pretty good, and divides people into groups based on the number of ...
+                <p>Matthew McMillan designs certificates using GIMP for competitions he organize since he has some graphic design skills. If you’d like him to do this for your competition, please let him know well in advance so he has time to do this. Oh, by the way, when you do get in touch with us, let us know what your favorite flavor of ice cream is. That lets us know you’re actually reading  ...
```
way too colloquial, can we make this more professional?

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="cubecomps">CubeComps</h3>
+                <p>CubeComps is the live results system currently used just about everywhere. This will need to be set up after the competition is announced. You should email <a href="mailto:admins@cubecomps.com">admins@cubecomps.com</a> after it’s announced to get this set up. There is also some setup work involved in actually importing your registration file, and you’ll need to set up rounds an ...
+                <p>When registration does close, you’ll need to go into the registration manager on the WCA website, and sort competitors by first name. Then, select all paid competitors and click on “Export to CSV”. You’ll need this to import into both Groupifier and CubeComps.</p>
+                <p>In CubeComps, after your email <a href="mailto:admins@cubecomps.com">admins@cubecomps.com</a> to create a site for your competition, you can log in as the administrator and upload the csv. You’ll need to set cutoffs for all of the events with cutoffs so that they are processed correctly. You’ll also need to set the number of advancing competitors. Let us know if you need help w ...
```
drop the animal thing please

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <p>When registration does close, you’ll need to go into the registration manager on the WCA website, and sort competitors by first name. Then, select all paid competitors and click on “Export to CSV”. You’ll need this to import into both Groupifier and CubeComps.</p>
+                <p>In CubeComps, after your email <a href="mailto:admins@cubecomps.com">admins@cubecomps.com</a> to create a site for your competition, you can log in as the administrator and upload the csv. You’ll need to set cutoffs for all of the events with cutoffs so that they are processed correctly. You’ll also need to set the number of advancing competitors. Let us know if you need help w ...
+                <h3 id="scorecards-and-certificates">Scorecards and certificates</h3>
+                <p>The organizer is responsible for generating and printing scorecards as well as designing and printing the certificates. <a href="https://jonatanklosko.github.io/groupifier/">Groupifier</a> is typically used for scorecard generation of first round cards and CubeComps is used to generate blank cards. Groupifier is pretty good, and divides people into groups based on the number of ...
```
drop "I'm" (and also drop reference to any future plans, prospective organizers don't need to know until it's ready)

This also gets off topic, can you move the thing about emailing competitors about parking to a separate section?

Also worth mentioning that they either need blank scorecards or access to a printer.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <p>The organizer is responsible for generating and printing scorecards as well as designing and printing the certificates. <a href="https://jonatanklosko.github.io/groupifier/">Groupifier</a> is typically used for scorecard generation of first round cards and CubeComps is used to generate blank cards. Groupifier is pretty good, and divides people into groups based on the number of ...
+                <p>Matthew McMillan designs certificates using GIMP for competitions he organize since he has some graphic design skills. If you’d like him to do this for your competition, please let him know well in advance so he has time to do this. Oh, by the way, when you do get in touch with us, let us know what your favorite flavor of ice cream is. That lets us know you’re actually reading  ...
+                <h3 id="sponsorship">Sponsorship</h3>
+                <p>Many competitions are sponsored by cube stores, who will provide prizes and sometimes even vend at the competition. You should try to figure this out before announcing.</p>
```
How do they figure this out?

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <p>The Delegate will create a WCA website for your competition and send you a link. Be sure you’re signed in to view the site, since it won’t be public yet. You are responsible for filling in all relevant information, and then the Delegate will approve the site and send it to the WCA Competition Announcement Committee when it’s ready to announce. One of the decisions you’ll have t ...
+                <h2 id="after-announcing-the-competition">After announcing the competition</h2>
+                <p>There is still work to be done after the competition is announced! During this time it is imperative that you regularly check your email and respond to competitors and the delegate as many things are time sensitive.</p>
+                <h3 id="registration-competitor-limits-and-wait-lists">Registration, competitor limits, and wait lists</h3>
```
inconsistency -- wait list vs waitlist

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="the-website">The website</h3>
+                <p>The Delegate will create a WCA website for your competition and send you a link. Be sure you’re signed in to view the site, since it won’t be public yet. You are responsible for filling in all relevant information, and then the Delegate will approve the site and send it to the WCA Competition Announcement Committee when it’s ready to announce. One of the decisions you’ll have t ...
+                <h2 id="after-announcing-the-competition">After announcing the competition</h2>
+                <p>There is still work to be done after the competition is announced! During this time it is imperative that you regularly check your email and respond to competitors and the delegate as many things are time sensitive.</p>
```
inconsistent -- delegate vs Delegate

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+        </div>
+        <div class="card-text">
+                <p>If you’re looking to organize a cube competition in New England, you’re in the right place! Please read this <strong>entire</strong> page before getting in touch with, as it’s important that you understand exactly what organizing involves.</p>
+                <p><a href="https://docs.google.com/spreadsheets/d/1-qjvGF3loVR0sr-AaDGZE2C7YgwcC1dP2wdPc3BT8fM/edit?usp=sharing">Here</a>’s a link to the competition organization spreadsheet. It will be discussed later on this page. Please go to this page, go to File &gt; Make a Copy, and rename it to your competition.</p>
```
Seems like the first sheet of that is mostly aimed at making the schedule for the WCA site, this isn't as necessary anymore since we have the scheduling tool, should this be updated?

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h2 id="role-of-the-organizer-and-the-delegate">Role of the organizer and the Delegate</h2>
+                <p>The organizer and Delegate have several roles, but these can sometimes vary by region throughout the world. Here we want to describe exactly what the Delegate will do, and what the organizer (you) will need to do.</p>
+                <h3 id="choosing-the-date">Choosing the date</h3>
+                <p>Competitions have to be about 3 weeks apart if they’re within 100km of each other by WCA Regulations. Once you start with the organizing process, the Delegate will share a spreadsheet with you with upcoming competitions so we can ensure your competition doesn’t conflict with other competitions. Of course, the date must also work for the Delegate, organizer(s), and the venue its ...
```
A thought...does this give the organizer too much of an impression that they get to pick the date?  Would it be better to replace this with something like "We'll work with you to pick a date that works for you, the Delegates' schedules, and the schedule of other competitions in the region"?  I don't love it when organizers email saying the day they want to hold a competition (and giving them more reason to believe they picked a good day by following the rules here, rather than talking to us, doesn't help).  I'd prefer the workflow of collaboratively picking the date while the delegate is responsible for ensuring it's a good day.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="choosing-the-date">Choosing the date</h3>
+                <p>Competitions have to be about 3 weeks apart if they’re within 100km of each other by WCA Regulations. Once you start with the organizing process, the Delegate will share a spreadsheet with you with upcoming competitions so we can ensure your competition doesn’t conflict with other competitions. Of course, the date must also work for the Delegate, organizer(s), and the venue its ...
+                <h3 id="the-venue">The venue</h3>
+                <p>The venue is a key part of the competition experience. The organizer is entirely responsible for choosing and booking the venue. Of course, the Delegate needs to approve the venue (do NOT book without explicit Delegate approval). In general, you should look for a venue that has</p>
```
Generally I'd prefer bold over capitalized ("NOT")

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="deciding-on-events">Deciding on events</h3>
+                <p>You definitely have some say over what events you want, but this competition should benefit the region too. Holding at least some beginner events, like 3x3, 2x2, and Pyraminx allows beginners to compete which is good for the community. It’s fine if you want to have less common events like 7x7, but the competition should be fun for everyone. In terms of scheduling, you should cr ...
+                <h3 id="the-schedule">The schedule</h3>
+                <p>The organizer is responsible for creating the schedule, and the Delegate will approve it and make suggestions to try to accurately reflect realistic times. Please fill out the scheduling section of the spreadsheet at the top of this document when it’s time to make a schedule. While this will vary greatly depending on the competition, you should almost always have a new competit ...
```
When is it time to make a schedule?

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="the-schedule">The schedule</h3>
+                <p>The organizer is responsible for creating the schedule, and the Delegate will approve it and make suggestions to try to accurately reflect realistic times. Please fill out the scheduling section of the spreadsheet at the top of this document when it’s time to make a schedule. While this will vary greatly depending on the competition, you should almost always have a new competit ...
+                <h3 id="the-website">The website</h3>
+                <p>The Delegate will create a WCA website for your competition and send you a link. Be sure you’re signed in to view the site, since it won’t be public yet. You are responsible for filling in all relevant information, and then the Delegate will approve the site and send it to the WCA Competition Announcement Committee when it’s ready to announce. One of the decisions you’ll have t ...
```
Can we just recommend specific timeframes?  e.g. 1 week after the competition is announced, and 1 week before the competition

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                        <li>lighting that is appropriate for solving that will be relatively consistent throughout the day</li>
+
+                </ul>
+                <p>When you start writing up a budget, you should have a good idea of how much the venue will cost. Schools and universities can often provide low-cost venues, but that’s not always an option. Be sure to make note of required custodians or police officers, as this usually costs extra. Places of worship like churches may be good candidates, but their space is often either too small ...
```
This paragraph starts with budget and ends with potential venue suggestions.  Keep on topic, move budget comments to the budget section.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <h3 id="the-finances">The finances</h3>
+                <p>The organizer is responsible for creating a budget, though the Delegate will need to approve it. The organizer will usually need to create a Stripe account to collect the registration fees and link this to the WCA website, but I’m happy to do this if that poses a problem. Please fill out the budget tab in the spreadsheet linked to at the top of this document when it’s time to m ...
+                <h3 id="deciding-on-events">Deciding on events</h3>
+                <p>You definitely have some say over what events you want, but this competition should benefit the region too. Holding at least some beginner events, like 3x3, 2x2, and Pyraminx allows beginners to compete which is good for the community. It’s fine if you want to have less common events like 7x7, but the competition should be fun for everyone. In terms of scheduling, you should cr ...
```
"Work with the delegate to select the events for the competition.  In general, at most competitions we recommend holding a few events like 3x3x3, 2x2x2, and Pyraminx that are popular with a large number of competitors"

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <p>We have a template of a Google Form we use to collect staff applications which the Delegate will set up. While we’re happy to provide guidance, the organizer is responsible for providing lunch to the staff and properly coordinating with all staff members.</p>
+                <p>Staff typically have perks of free registration and free lunch in exchange for helping out all day. Staff must pay to register and will be refunded after the competition, as an incentive to actually do what is expected of them. Staff applications are posted on the website to allow anyone to fill out a form and apply for staff, but it’s up to our discretion to vet these people a ...
+                <p>Additionally, at recent competitions, we have been doing what is known as the “Hungarian System” where all competitors are assigned judging roles. This helps people become familiar with judging, and reduces the need for asking for volunteers constantly. It’s great to have a competitor tutorial at the beginning of the competition to help people understand how judging works, and  ...
+                <h3 id="the-day-of">The day of</h3>
```
"During the competition"

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                <p>Staff typically have perks of free registration and free lunch in exchange for helping out all day. Staff must pay to register and will be refunded after the competition, as an incentive to actually do what is expected of them. Staff applications are posted on the website to allow anyone to fill out a form and apply for staff, but it’s up to our discretion to vet these people a ...
+                <p>Additionally, at recent competitions, we have been doing what is known as the “Hungarian System” where all competitors are assigned judging roles. This helps people become familiar with judging, and reduces the need for asking for volunteers constantly. It’s great to have a competitor tutorial at the beginning of the competition to help people understand how judging works, and  ...
+                <h3 id="the-day-of">The day of</h3>
+                <p>The Delegate will bring equipment to the competition, but you should budget for battery money as the equipment requires lots of batteries. The Delegate will usually arrive early to help set up, but it’s ideal if the venue could be set up the night before. Note that the organizer is expected to be there well before the start of the competition. The organizer should also be the o ...
```
Batteries need to be dealt with before the competition.  Mention in budget and equipment, not this section.

timreyn replied:
Expand on the responsibilities of the organizer during the competition, maybe a few bullet points rather than just a paragraph list.  This is often neglected by new organizers.

timreyn commented on src/templates/newengland-organizing.html:
```diff
@@ -0,0 +1,102 @@
+                </blockquote>
+                <p>– <em><a href="https://cubingusa.org/supported">CubingUSA</a></em></p>
+                <p>The main benefit of this is that you can use their nonprofit status (which can result in venue discounts) and that finances are handled in a more official way. If this is something you’re interested in, let us and we can work to make this happen. Note that this doesn’t change what the competition is actually like the day of.</p>
+                <h3 id="coordinating-staff">Coordinating Staff</h3>
```
This header has capitalized words, others don't.  Be consistent (probably capitalize everywhere)

timreyn reviewed (commented):
Thanks Matthew!

High level -- this is generally too wordy and colloquial.  If it's going to be on the cubingusa site I'd like it to be more formally written.  I left specific comments below, but some general things:

- make sure things stay on topic; if paragraphs start to stray from their header (as happens in a few places) then the document becomes harder to follow
- watch out for colloquial phrases like "stuff", "just", "really", "of course", "this will do"
- adverbs tend to make things more colloquial -- do a pass through looking for any adverbs that aren't actually adding anything (so things like "manually accepted" are fine, but "you definitely have" aren't necessary)
- there's various missing words in places, please do another pass looking for those

Left a bunch of specific comments but please also apply the thoughts to the rest of the document, I haven't commented on every example of bad patterns.

UnsolvedCypher committed:
Professionalize and clean up

UnsolvedCypher commented:
@timreyn Thank you for the review. I've gone through and cleaned up a lot of the tone and colloquialisms and incorporated most of your suggestions. Please let me know what you think. If you prefer to read this in Markdown, please see [here](https://docs.google.com/document/d/1lDTgQFySkILta7kYyp_JLg1nHjkfscZfRTQ3HHl1u9I/edit?usp=sharing).

jpbulman commented:
Staging: https://mattschanges-dot-staging-cubingusa-org.appspot.com./newengland

UnsolvedCypher commented:
@timreyn Would you mind taking a look at the revisions (see above) and letting me know what you think?

jpbulman commented:
Bump

timreyn commented:
Now that we have a Cube New England website I think that's a better place to collect New England specific stuff.
